.github/workflows/static.yml

```diff
@@ -0,0 +1,80 @@
+on: [pull_request]
+name: Static analysis
+
+jobs:
+  phpstan:
+    name: PHPStan
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: 8.0
+          coverage: none
+          tools: phpstan:0.12.92, cs2pr
```

Comment:
What about required PHPStan as dev dependencies? This way we can run it locally before pushing.

Reply:
Hard no. :)

Static analyzers are not a dependency of this project. It is just a tool to help you develop. Just like PHPstorm is a tool that will help you develop. 

Experienced developers can run these tools (installed globally) before they push. If you don’t, you will get an automated PR comment of things you should change. 

We could add a Makefile script to download and run static analyzers. I think that is overkill, but anyways it is outside the scope of this PR.

Reply:
Alright, I just wanted your point of view about this, since I'm in the "install locally to the tooling dependencies" team.

Thanks! And nice job as usual :)

Reply:
Yeah. I definitely see the value of your "team" when php-cs-fixer updated to a new major version or if your project is doing something psalm-specific. 

But I still find this to be the simplest, most automated way forward. Especially for small bundles/libraries like this one.